grunt/tasks/less.js

```diff
@@ -32,6 +32,25 @@ module.exports = function(grunt) {
 						imports+= "@import '" + trimmed + "';\n";
 					});	
 				}
+
+				if (options.config) {
+					var screenSize = {
+			                        "small": 520,
+			                        "medium": 760,
+			                        "large": 900
+					};
+					try {
+						var configjson = JSON.parse(grunt.file.read(options.config).toString());
+						screenSize = configjson.screenSize || screenSize;
+					} catch (e) {}
+
+					console.log(screenSize);
```

Comment:
If you're going to keep the console.log in, consider changing it to:
```
console.log("screen size: " + screenSize);
```
To give a bit more information about what the data being logged is